set.mm

```diff
@@ -70681,6 +70681,13 @@ Finite induction (for finite ordinals)
       FUGRRDUHUI $.
   $}
 
+
+$(
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+  Relations and functions (cont.)
+=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+$)
```

Comment:
That is a very good idea! It was confusing that theorems like dmexg etc. were contained in a section called "Finite induction (for finite ordinals)".